src/remote/playwrightClient.ts

```diff
@@ -32,15 +32,23 @@ export class PlaywrightClient {
     const { wsEndpoint, timeout = 30000 } = options;
     const connection = new Connection();
     const ws = new WebSocket(wsEndpoint);
-    connection.onmessage = message => ws.send(JSON.stringify(message));
+    connection.onmessage = message => {
+      if (ws.readyState === 2 /** CLOSING */ || ws.readyState === 3 /** CLOSED */)
```

Comment:
I think this can actually happen, and websocket is going to throw by itself. Should we reverse the check instead?

Reply:
The `ws` library does only throw if you pass a callback to the `send` method and/or promisify it which ends up like a silent faillure with the current state: 
https://github.com/websockets/ws/blob/055949fd23b0d7f6a23ba9b9532b7834909df192/lib/websocket.js#L940-L946

Reply:
> What is a use-case for that it can happen?

This code does not use `makeWaitForNextTask` currently, and it should. Once it does that, I _think_ we can end up dispatching a message after ws closes because we introduce asynchrony. I'd better be safe here and not throw.

Reply:
okay let me revert that part.